drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -54,6 +54,8 @@ struct mlx5_vdpa_caps {
 struct virtq_info {
 	uint32_t               rqn;
 	struct mlx5dv_devx_obj *rq_obj;
+	uint32_t                virtq_id;
```

Comment:
what would be the use of virtq_id?

Reply:
For modifying the VIRTQ object (for example when in LM you will want to change the ctrl_mkey).
devx SW in mlx5_ib driver expects you to put the obj_id inside the command.